### PR TITLE
doc review: traffic split qs

### PR DESCRIPTION
**Pull request by MelissaFlinn**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by MelissaFlinn** [comment]
@israel-hdez Here is a copy edit for your review - thank you...

File: docs/quickstart-traffic-splitting.md [hunk 1]
```diff
@@ -1,35 +1,24 @@
...
+  - On OpenShift, use the [deploy.ossm.sh](../scripts/deploy.ossm.sh) script to quickly install OpenShift Service Mesh (OSSM). On Red Hat OpenShift for AWS, you must uncomment a few lines in the `deploy.ossm.sh` script before you run it.
 
-If you are using another Kubernetes flavor (or also OpenShift), you can
-use [Istio](https://istio.io/). Use the installation method that better suits
-your needs. You must install `istiod` and `istio-ingressgateway` components. If
-you are
-[installing with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/),
-installing with the command `istioctl install` will provide a setup with these
-components.
+  - On another type of OpenShift (or Kubernetes), use [Istio](https://istio.io/) to install OSSM. You must also install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), use the `istioctl install` command which also installs `istiod` and `istio-ingressgateway`.
```
Comment (israel-hdez) [inline docs/quickstart-traffic-splitting.md:13]: Istio is the upstream project of OSSM. So, we cannot say that Istio is used to install OSSM. We just install Istio and that's it. Istio works in (almost?) any Kubernetes system (including OpenShift), while OSSM works only in OpenShift.  The `istiod` and `istio-ingressgateway` components are part of Istio and can be installed selectively.

Additionally, I just remembered that this repository is OpenShift specific and not meant for any other Kubernetes system.

I tried to fix this second bullet to denote all this, but I think my suggestion needs you refinement:

```suggestion
  - Alternatively, use [Istio](https://istio.io/). You must install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), using the `istioctl install` command includes the required `istiod` and `istio-ingressgateway` components.
```

File: docs/quickstart-traffic-splitting.md [hunk 2]
```diff
@@ -48,35 +37,25 @@ Once the Service Mesh is installed:
         - '*'
         port:
           name: http
-          number: 80
+          number: 8
```
Comment (israel-hdez) [inline docs/quickstart-traffic-splitting.md:40]: This seems to be a change by mistake. Please, fix.

File: docs/quickstart-traffic-splitting.md [hunk 3]
```diff
@@ -48,35 +37,25 @@ Once the Service Mesh is installed:
...
-(as mentioned in the [Prerequisites](#prerequisites)) you will deploy models in
-the `modelmesh-serving` namespace, because this is the namespace where the
-ServingRuntimes are available.
+    # Deploy odh-model-controller
+    make deploy -e IMG=quay.io/edgarhz/odh-model-controller:service-mesh-integration
+    ```
 
-1. If you are using OSSM, enrol both the `opendatahub` and
-   this `modelmesh-serving` namespaces in the mesh:
+1. Enroll both the `opendatahub` and this `modelmesh-serving` namespaces in the mesh:
```
Comment (israel-hdez) [inline docs/quickstart-traffic-splitting.md:58]: This is only for OSSM. For Istio, this step is not done. So, I think we should keep the `If you are using OSSM,` text (or some more elegant writing)

**Comment by MelissaFlinn** [comment]
Thank you @israel-hdez 
I addressed your comments - this is now Ready for merge after other necessary reviews.

**Review by israel-hdez — APPROVED** [review]
Thanks, @MelissaFlinn !

**Pull request merged**